src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
+const table = document.querySelector('.field');
+const tbody = table.querySelector('tbody');
+const appendColum = document.querySelector('.append-column');
```

Comment:
The variable name `appendColum` is misspelled. It should be `appendColumn` to reflect the correct spelling.